src/main/java/fi/hsl/pulsar/mqtt/MessageProcessor.java

```diff
@@ -151,4 +156,18 @@ public boolean isMqttConnected() {
         }
         return false;
     }
+
+    public boolean lastMsgReceivedTimeHealthy() {
+        if (HEALTHY_MSG_LAST_RECEIVED_SECS == -1) {
```

Comment:
If env variable HEALTHY_MSG_LAST_RECEIVED_SECS is unset, the test is disabled (returns always true).